tests/test_backup.py

```diff
@@ -439,3 +444,83 @@ def test_positive_backup_offline_all(ansible_module):
     for result in contacted.values():
         logger.info(result['stdout'])
         assert "FAIL" not in result['stdout']
+
+
+@capsule
+def test_negative_backup_online_nodir(ansible_module):
+    """Take online backup of server with no destination
+
+    :id: 298176a7-ba1d-4fcd-9718-d62c47900bb5
+
+    :setup:
+
+        1. foreman-maintain should be installed.
+    :steps:
+        1. Run foreman-maintain backup online
```

Comment:
just nitpick, can you make it `Run foreman-maintain backup online without providing directory`